packages/desktop/components/LatestMatchMonitor/index.tsx

```diff
@@ -27,6 +34,15 @@ export const LatestMatchMonitor = () => {
               Logs are NOT being automatically uploaded to WoW Arena Logs!
             </div>
           )}
+          {diskSpaceRemaining > -1 && (
+            <div className="text-2xl font-bold text-red-400 badge badge-lg badge-error p-5">
```

Comment:
might want to try https://daisyui.com/components/alert/ and put at the top of the page as a standard way for warnings since we now have quite a few